src/scores/probability/quantile_loss_impl.py

```diff
@@ -0,0 +1,101 @@
+"""
+Implementation of quantile loss (score)
+"""
+from typing import Optional, Sequence
+
+import dask.array as da
```

Comment:
Dask isn't a core dependency, it's an optional add one and included in the dev dependencies. I think it might be able to implement this without using `dask.array` like how the other scores are implemented.